apps/ewallet/lib/ewallet/web/v1/event_handlers/event.ex

```diff
@@ -21,15 +21,21 @@ defmodule EWallet.Web.V1.Event do
 
     Enum.each(topics, fn topic ->
       Enum.each(endpoints(), fn endpoint ->
-        endpoint.broadcast(
-          topic,
-          event,
-          payload
-        )
+        broadcast_if_compiled(endpoint, topic, event, payload)
       end)
     end)
   end
 
+  defp broadcast_if_compiled(endpoint, topic, event, payload) do
+    if Code.ensure_compiled?(endpoint) do
```

Comment:
What cases would make this not compiled?

Reply:
Oh I forgot I actually moved the check here! You can discard my comment above, we can remove the `ensure_compiled?` from the config files. The explanation is however the same one:

```
# We use 'Code.ensure_compiled?' to ensure the endpoints are loaded.
# When running tests from inside the Admin API/eWallet API/eWallet sub apps,
# those endpoint modules are not available and therefore should not be added to
# the list of endpoints that needs to be notified. If they are and the
# Event emitter tries to call them, it will result in a 500 error.
```